src/constants.js

```diff
@@ -13,4 +13,12 @@ export const envs = {
 
 // Variables
 export const prefix = '++';
-export const validCmds = ['ama', 'faq', 'pins', 'salary', 'settings', 'perms'];
+export const validCmds = [
```

Comment:
Can you handle this like a java enum?  I.e. map each valid command to its respective bot/functionality.

Reply:
In this case, I'm not sure it'd work and might just overcomplicate things. We need the string specifically. We could make them like string enums but seems unnecessary right now

Reply:
We could think about this more but outside of the scope of this PR